src/common/domain/openInZeplin/flow/openInZeplinFlow.ts

```diff
@@ -0,0 +1,61 @@
+import * as vscode from "vscode";
```

Comment:
This is not a new file. It's relocated and a little bit changed. Previous version:
https://github.com/zeplin/vscode-extension/blob/3acb791eca73e7957c5ed51167b72e8ccbc55691/src/sidebar/openInZeplin/flow/openInZeplinFlow.ts